src/functions-and-arrays.js

```diff
@@ -25,14 +100,74 @@ function sum() {}
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
+let avg = 0
+function averageNumbers(numbersAvg) {
+
+
+   if(numbersAvg.length === 0) {
+    return null
+
+  }
+  if(numbersAvg.length === 1) {
+    return numbersAvg[0]
+
+  }
+
+
+  for(let i=0; numbersAvg.length; i++ ){
+
+     avg = avg + numbersAvg[i]
+
+  }
+  avg = avg / numbersAvg.length
+
+  return avg
+
+
+}
+
 
-function averageNumbers() {}
 
 
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
+let wordLength = 0
+let avg2 = 0
+let avgOneWOrd =0
+function averageWordLength(wordsArr) { 
```

Comment:
```suggestion
sumNumbers()
```
En esta podrías haber utilizado tu función sumNumbers creada previamente para no tener que volver a sumar, pero es un extra que te doy para ahorrar tiempo, si hay una función que ya hace algo que necesitas volver a hacer. Aprovechala!